proteus/mprans/cRANS3PSed.cpp

```diff
@@ -0,0 +1,31 @@
+#include "pybind11/pybind11.h"
+#include "pybind11/stl_bind.h"
+
+#define FORCE_IMPORT_ARRAY
+#include "RANS3PSed.h"
+
+#if defined(__GNUC__) && !defined(__clang__)
+    namespace workaround
+    {
+        inline void define_allocators()
+        {
+            std::allocator<int> a0;
+            std::allocator<double> a1;
+        }
+    }
+#endif
+
+namespace py = pybind11;
+using proteus::cppRANS3PSed_base;
+
+PYBIND11_MODULE(cRANS3PSed, m)
+{
+    xt::import_numpy();
+
+    py::class_<cppRANS3PSed_base>(m, "cppRANS3PSed_base")
+        .def(py::init(&proteus::newRANS3PSed))
+        .def("calculateResidual", &cppRANS3PSed_base::calculateResidual)
+        .def("calculateJacobian", &cppRANS3PSed_base::calculateJacobian)
+        .def("calculateVelocityAverage", &cppRANS3PSed_base::calculateVelocityAverage);
+
```

Comment:
This file looks like a duplicata of `RANS3PSed.cpp` or am I missing something? If this is a duplicate, this defines the module twice. Also I would be in favor of keeping the name of the implementation file consistent with that of the header file. The 'c' prefix makes me think of a file generated by cython while it is not.